enterprise/models.py

```diff
@@ -852,6 +852,54 @@ def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
 
 
 @python_2_unicode_compatible
+class EnterpriseCatalog(TimeStampedModel):
```

Comment:
On second thought, let's call this `EnterpriseCustomerCatalog` since it better reflects what this model actually is. You also won't have to refactor the names of the existing views/serializers.